src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -12,6 +12,15 @@
       Background="Transparent"
       mc:Ignorable="d">
 
+    <Page.Resources>
+        <!--  Info Bar  -->
+        <Style x:Key="SettingsInfoBarStyle"
+               TargetType="mux:InfoBar">
+            <Setter Property="Margin" Value="-2" />

```

Comment:
For some reason there's a bit of space all around the info bar that the background will not cover (and is transparent), hence the negative margin

Reply:
We should probably file an issue upstream that Informational `InfoBar`s have a transparent BG

Reply:
In [Controls v1, an `InfoBar`](https://github.com/microsoft/microsoft-ui-xaml/blob/9052972906c8a0a1b6cb5d5c61b27d6d27cd7f11/dev/InfoBar/InfoBar_themeresources_v1.xaml#L13) has either `#2f2f2f` or `#2B2B2B` as the background, which are both fully opaque

In In [Controls v2, an `InfoBar`](https://github.com/microsoft/microsoft-ui-xaml/blob/775568ceb6ef0821b27b49b0b16f92270e8df65f/dev/InfoBar/InfoBar_themeresources.xaml#L14) uses [`SystemFillColorAttentionBackgroundBrush`](https://github.com/microsoft/microsoft-ui-xaml/blob/5673d0ca4b6b0e4645637bbab37f48f4825baec3/dev/CommonStyles/Common_themeresources_any.xaml#L100) which is `#08FFFFFF`, so it's partially transparent (mostly transparent actually).

Reply:
Ah it's already in discussion... https://github.com/microsoft/microsoft-ui-xaml/issues/5741

Reply:
I would rather give the thing that hosts the infobar a background than weirdly expand its margins.......

Reply:
(AKA: this feels like a hack)